showcase-docs/getting-started.md

```diff
@@ -2,6 +2,81 @@
+
+- **Anonymized-focused configuration**:
+
+  - IP addresses are anonymized (`maskIP: true`), and recorded as `0.0.0.0`.
+  - `anonymousId` used for tracking is a hash derived from the user's username.
+
+- **Data Collection Overview**:
+  - **Events Tracked**: Page visits, clicks on links or buttons.
+  - **Common Data Points for All Events**:
+    - User-related info: Locale, timezone, userAgent (browser and OS details).
+    - Page-related info: Title, Category, Extension name, URL, path, referrer, search parameters.
+
+This ensures a thorough understanding of user interactions with the application while maintaining user anonymity and privacy.
+
+The data will be used only for internal analysis and product improvement.
+
+If you wish to enable telemetry data collection, follow the steps below.
+
+### Enable Telemetry Using Helm Chart
+
+To turn on the telemetry while using the Helm chart, you need to enable Segment provider plugin in your Helm `values.yaml` file by adding the following configuration:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: false
+```
+
+By default, the Segment plugin is configured to send data to Red Hat. If you wish to change the destination, you can do so by setting the `SEGMENT_WRITE_KEY` environment variable to the desired Segment write key.
+
+```yaml
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: SEGMENT_WRITE_KEY
+        value: <segment_key>
+```
+
+If you wish to subsequently disable telemetry data collection, use one of the following methods described below.
+
+### Disable Telemetry Using Helm Chart
+
+To turn off Segment telemetry while using the Helm chart, you need to deactivate the Segment provider plugin in your Helm `values.yaml` file by adding the following configuration:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: true
+```
+
+### Disable Telemetry for Local Development
+
+When running locally without using the `dynamic-plugins.default.yaml` file, the Segment plugin is not activated by default.
+However, if your configuration is based on `dynamic-plugins.default.yaml`, you can disable the Segment plugin by adding the below lines to your configuration file:
+
+```yaml
+dynamicPlugins:
+  plugins:
+    - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+      disabled: true
+```
+
+Afterward, remove the Segment plugin directory by deleting `dynamic-plugins-root/janus-idp-backstage-plugin-analytics-provider-segment`
+
+### Disabling Telemetry in Continuous Integration (CI) Environments
+
+To disable telemetry while running Backstage in a CI environment, set the `SEGMENT_TEST_MODE` environment variable to `true`. This action deactivates telemetry transmissions.
```

Comment:
@kadel, Can we mention the file name in which we have to set this environmet variable?

Reply:
This depends on what CI system is used. For some CI systems, it will be in a file. For others, this could be somewhere in the web UI. People running this in CI will know where to configure this in their system